controllers/rollingupgrade_controller.go

```diff
@@ -73,7 +73,7 @@ func (r *RollingUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	err := r.Get(ctx, req.NamespacedName, rollingUpgrade)
 	if err != nil {
 		if kerrors.IsNotFound(err) {
-			r.AdmissionMap.Delete(req.NamespacedName)
+			r.AdmissionMap.Delete(rollingUpgrade.NamespacedName())
```

Comment:
This will be probematic, if `IsNotFound` is true rollingUpgrade will be nil, best keep it `req.NamespacedName`

Reply:
Downstream from this, it's OK to use `rollingUpgrade.NamespacedName()`